src/main/resources/application.yml

```diff
@@ -17,6 +17,17 @@ spring:
   data:
     mongodb:
       uri: mongodb://${DB_USER:admin}:${DB_PASSWORD:pwd}@${DB_HOST:localhost}:${DB_PORT:27017}/${DB_NAME:sync}?authSource=${AUTH_SOURCE:admin}
+  redis:
+    host: ${REDIS_HOST:localhost}
+    port: ${REDIS_PORT:6379}
+    user: ${REDIS_USER:default}
+    password: ${REDIS_PASSWORD:password}
+    timeout: 60000
+    database: 0
+    time-to-live: ${REDIS_DATA_TTL:2}
```

Comment:
The data ttl should probably default to a long time; this was just for local test purposes. Also the ttls are interpreted as minutes, while the timeout is seconds - it might be worth making the units consistent to avoid confusion.